confluent_kafka_helpers/loader.py

```diff
@@ -51,18 +55,43 @@ def find_duplicated_messages(messages, logger=logger):
             )
 
 
+def default_error_handler(kafka_error):
+    code = kafka_error.code()
+    if code == KafkaError._PARTITION_EOF:
+        logger.debug("Reached end of partition")
+        raise EndOfPartition
+    elif code == KafkaError._TRANSPORT:
+        statsd.increment(f'{base_metric}.loader.message.count.error')
+        raise KafkaTransportError(kafka_error)
+    else:
+        statsd.increment(f'{base_metric}.loader.message.count.error')
+        raise KafkaException(kafka_error)
+
+
 class MessageGenerator:
-    def __init__(self, consumer, key, key_filter):
+    def __init__(
+        self, consumer, key, key_filter,
+        error_handler: Callable = default_error_handler
+    ) -> None:
         self.consumer = consumer
         self.key = key
         self.key_filter = key_filter
-        self.messages = []
+        self.messages: List = []
```

Comment:
What's the reason for adding the Type here?

Reply:
@ataz Mypy complained...